packages/webapp/src/locales/en-US/requests.json

```diff
@@ -102,6 +102,7 @@
     "_assignSpecialistPlaceholder.comment": "Placeholder text for Assign Specialist",
     "addTagPlaceholder": "Add tag...",
     "_addTagPlaceholder.comment": "Placeholder text for add tag",
+    "description": "Description",
```

Comment:
You should add another line underneath with an underscore (`_description`) explaining what your field is doing. This is used by the translators to create the other languages files.